VideoLocker/res/values/styles.xml

```diff
@@ -306,23 +307,24 @@
         <item name="android:paddingTop">@dimen/vertical_input_padding</item>
     </style>
 
-    <style name="edX.Widget.Button" parent="@android:style/Widget.TextView">
```

Comment:
Yup, my mistake. Mirza also filed that here: https://openedx.atlassian.net/browse/MA-1858